Patch - enforce line endings for platform-specific gradle scripts.

anenviousguest opened the pull request:
This tiny change improves the ergonomics - it enforces line endings for the Gradle build scripts which _are_ platform dependent.

`gradlew` is a shell script and must have LF line ending.
`gradlew.bat` is a windows CMD file and must have CRLF line ending.

markbrockhoff commented:
Thanks for taking care of the Windows dev experience of this plugin, really appreciated.
Just one question: Don't you want to re-add gradlew.bat with windows line endings? It seems to me like right now you only added the gitattributes file without updating the git cache for the file :)

anenviousguest commented:
Fair enough @markbrockhoff !

I've updated the line endings, here is the output from my WSL (linux) console:

```
➜  spectral-intellij-plugin git:(main) git ls-files --eol gradlew*
i/lf    w/lf    attr/                   gradlew
i/lf    w/lf    attr/                   gradlew.bat
➜  spectral-intellij-plugin git:(main) git checkout fix/win-settings
Switched to branch 'fix/win-settings'
Your branch is up to date with 'origin/fix/win-settings'.
➜  spectral-intellij-plugin git:(fix/win-settings) ✗ git ls-files --eol gradlew*
i/lf    w/lf    attr/text eol=lf        gradlew
i/crlf  w/crlf  attr/text eol=crlf      gradlew.bat
```
